src/devices/src/virtio/fs/linux/passthrough.rs

```diff
@@ -665,8 +665,8 @@ fn forget_one(
             // Synchronizes with the acquire load in `do_lookup`.
             if data
                 .refcount
-                .compare_and_swap(refcount, new_count, Ordering::Release)
-                == refcount
+                .compare_exchange(refcount, new_count, Ordering::Release, Ordering::Release)
```

Comment:
According to [this](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.compare_and_swap), the fourth argument should be `Ordering::Relaxed`.

Reply:
Resolved in latest commit.